astropy/utils/xml/check.py

```diff
@@ -55,9 +55,9 @@ def check_mime_content_type(content_type):
     (syntactically at least), as defined by RFC 2045.
     """
     ctrls = ''.join(chr(x) for x in range(0, 0x20))
-    token_regex = '[^()<>@,;:\\\"/[\]?= %s\x7f]+' % ctrls
+    token_regex = '[^()<>@,;:\\\"/[\]?= {}\x7f]+'.format(ctrls)
```

Comment:
Touching regex stuff always makes me nervous but I guess if this change messed things up, we would have caught that in the regression tests. So, I guess this is okay.